Broken Links Found: Moonbeam

### 🚨 Link Checker Report
  **Project:** Moonbeam
  **Date:** 2026-01-14
  **Workflow Run:** https://github.com/papermoonio/workflows/actions/runs/20993635348

  ---
  # Summary

| Status        | Count |
|---------------|-------|
| 🔍 Total      | 38940 |
| ✅ Successful | 38351 |
| ⏳ Timeouts   | 0     |
| 🔀 Redirected | 0     |
| 👻 Excluded   | 587   |
| ❓ Unknown    | 0     |
| 🚫 Errors     | 2     |

## Errors per input

### Errors in moonbeam-mkdocs/site/learn/features/governance/index.html

* [ERROR] [https://wiki.polkadot.com/general/web3-and-polkadot/#fellowship](https://wiki.polkadot.com/general/web3-and-polkadot/#fellowship) | Network error: error sending request for url (https://wiki.polkadot.com/general/web3-and-polkadot/#fellowship) Maybe a certificate error?

### Errors in moonbeam-mkdocs/site/builders/integrations/oracles/dia/index.html

* [ERROR] [https://www.diadata.org/docs/nexus/reference/smart-contracts/diaoraclev2.sol#diaoraclev2-sol](https://www.diadata.org/docs/nexus/reference/smart-contracts/diaoraclev2.sol#diaoraclev2-sol) | Network error: error sending request for url (https://www.diadata.org/docs/nexus/reference/smart-contracts/diaoraclev2.sol#diaoraclev2-sol) Maybe a certificate error?
[Full Github Actions output](https://github.com/papermoonio/workflows/actions/runs/20993635348?check_suite_focus=true)

--- a/learn/features/governance.md
+++ b/learn/features/governance.md
@@ -300,7 +300,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-basics.txt
+++ b/llms-files/llms-basics.txt
@@ -2327,7 +2327,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-dev-environments.txt
+++ b/llms-files/llms-dev-environments.txt
@@ -4355,7 +4355,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-ethereum-toolkit.txt
+++ b/llms-files/llms-ethereum-toolkit.txt
@@ -18502,7 +18502,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-gmp-providers.txt
+++ b/llms-files/llms-gmp-providers.txt
@@ -2946,7 +2946,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-governance.txt
+++ b/llms-files/llms-governance.txt
@@ -335,7 +335,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 
@@ -3109,7 +3110,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-indexers-and-queries.txt
+++ b/llms-files/llms-indexers-and-queries.txt
@@ -4351,7 +4351,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-integrations.txt
+++ b/llms-files/llms-integrations.txt
@@ -2771,7 +2771,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-json-rpc-apis.txt
+++ b/llms-files/llms-json-rpc-apis.txt
@@ -3812,7 +3812,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-libraries-and-sdks.txt
+++ b/llms-files/llms-libraries-and-sdks.txt
@@ -6335,7 +6335,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-node-operators-and-collators.txt
+++ b/llms-files/llms-node-operators-and-collators.txt
@@ -5793,7 +5793,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-oracle-nodes.txt
+++ b/llms-files/llms-oracle-nodes.txt
@@ -2939,7 +2939,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-precompiles.txt
+++ b/llms-files/llms-precompiles.txt
@@ -11307,7 +11307,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-staking.txt
+++ b/llms-files/llms-staking.txt
@@ -2825,7 +2825,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-substrate-toolkit.txt
+++ b/llms-files/llms-substrate-toolkit.txt
@@ -4290,7 +4290,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-tokens-and-accounts.txt
+++ b/llms-files/llms-tokens-and-accounts.txt
@@ -4466,7 +4466,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-tutorials.txt
+++ b/llms-files/llms-tutorials.txt
@@ -2709,7 +2709,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-xc-20.txt
+++ b/llms-files/llms-xc-20.txt
@@ -5223,7 +5223,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-xcm-remote-execution.txt
+++ b/llms-files/llms-xcm-remote-execution.txt
@@ -5693,7 +5693,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-files/llms-xcm.txt
+++ b/llms-files/llms-xcm.txt
@@ -6839,7 +6839,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

--- a/llms-full.txt
+++ b/llms-full.txt
@@ -38595,7 +38595,8 @@ In addition, there is a Kill Origin, which is for bad referenda intending to har
 
 ### Rights of the OpenGov Technical Committee {: #rights-of-the-opengov-technical-committee }
 
-On Polkadot, the Technical Committee from Governance v1 was replaced with the Fellowship, which is a "mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Kusama and/or Polkadot networks and protocol," according to [Polkadot's wiki](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}.
+
+On Polkadot, the Technical Committee from Governance v1 was replaced by the [Fellowship](https://wiki.polkadot.com/learn/learn-polkadot-technical-fellowship/){target=\_blank}, a self-governing expert body composed of members with deep technical knowledge of the Kusama and Polkadot networks protocols.
 
 For Moonbeam's implementation of OpenGov, instead of the Fellowship, there is a community OpenGov Technical Committee that has very similar power to that of the Fellowship. Their power in governance resides in their ability to whitelist a proposal. OpenGov Technical Committee members may only vote to whitelist a proposal if whitelisting that proposal would protect against a security vulnerability to the network. The passing threshold of the OpenGov Technical Committee members on whether to whitelist a proposal is determined by governance. As such, the OpenGov Technical Committee has very limited power over the network. Its purpose is to provide technical review of urgent security issues that are proposed by token holders.
 

